yt_dlp/extractor/youtube.py

```diff
@@ -3703,7 +3703,8 @@ def append_client(*client_names):
                 pr_video_id = traverse_obj(pr, ('videoDetails', 'videoId'))
                 if pr_video_id and pr_video_id != video_id:
                     self.report_warning(
-                        f'Skipping player response from {client} client (got player response for video "{pr_video_id}" instead of "{video_id}")' + bug_reports_message())
+                        f'Skipping player response from {client} client (got player response for video "{pr_video_id}" instead of "{video_id}")')
+                    last_error = ValueError("Could not extract any player response (perhaps your IP is being blocked by Youtube?)")
```

Comment:
IMO we shouldn't raise like this; we should instead fix #7715, the format selection bug that is causing `best` to select storyboards as a fallback, and then we will properly raise for no formats instead.

we could add the IP block note to the warning though, something like this?
```suggestion
                        f'Skipping player response from {client} client'
                        + f'(got player response for video "{pr_video_id}" instead of "{video_id}" possibly due to IP address ban by Youtube)')
```

Reply:
okay so you say if we get no prs then we should handle that externally through the fix for #7715? I guess the only time we get faulty prs would be when we are blocked, so adding it to the warning would work. However, if you query a lot of prs, getting the same *lengthy* warning over and over again might not be great...

Reply:
I like the error, but your implementation shows it even if only some PRs are skipped

Reply:
Right, bc we report warning if we got some responses back... Ill try and adjust it